### PR TITLE
[LRN] Add defensive CSS for hidden textarea

### PR DESCRIPTION
**Pull request by karoltarasiuk**

### CODE REVIEW

File: src/css/textarea.less [hunk 1]
```diff
@@ -6,6 +6,11 @@
 
     // TODO: why is this here?
     .user-select(text);
+
+   > textarea {
+     opacity: 0; // Defense against iOS cursor appearing in client pages
```
Comment (laughinghan) [inline src/css/textarea.less]: Is this to fix #310? Should be fixed already (https://github.com/mathquill/mathquill/pull/554/files#diff-f00a1908060a90535e10dbd689e744ddR22)

**Comment by laughinghan** [comment]
Also, we prefer to have these in `.mq-textarea *` rather than `.mq-textarea > textarea` because we let people substitute something else with the `substituteTextarea` config option. (People substitute stuff like spans with a tabindex attribute (which makes it focusable) in order to disable the virtual touchscreen keyboard.)

File: src/css/textarea.less (hunk 1 above)
Comment (karoltarasiuk) [inline src/css/textarea.less]: hey @keremdaldal, do you remember whether this is the very issue?

Comment (keremdaldal) [inline src/css/textarea.less]: @karoltarasiuk, @laughinghan, yes it's the same issue. You can safely remove this line.

File: src/css/textarea.less [hunk 2]
```diff
@@ -6,6 +6,11 @@
...
+     overflow: hidden; // Defense against bug on Chrome with retina
```
Comment (karoltarasiuk) [inline src/css/textarea.less]: @keremdaldal do you remember what was this one?

File: src/css/textarea.less (hunk 1 above)
Comment (karoltarasiuk) [inline src/css/textarea.less]: good stuff thanks! I will update PR, once we solve the other one

**Comment by karoltarasiuk** [comment]
Spoken to @keremdaldal and we decided to close it as it seems those fixes are not needed anymore.

**Pull request closed without merging**